packages/terra-consumer-nav/src/Nav.scss

```diff
@@ -24,7 +24,7 @@
     position: fixed;
     width: var(--terra-consumer-profile-link-width, 320px);
 
-    @supports (-webkit-overflow-scrolling: touch) {
+    @media screen and (max-width: $terra-medium-breakpoint) {
```

Comment:
this change is complicated, basically we always want the profile bar static on mobile menu...

Reply:
@amf9t2 ^^